code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -297,7 +297,7 @@
 		..()
 		new /obj/item/clothing/accessory/badge/holo/detective(src)
 		new /obj/item/clothing/gloves/black(src)
-		//new /obj/item/gunbox(src) // Vorestation edit. If you can't be trusted to not be stupid with lethals, you can't be trusted with less-than-lethals either.
+		new /obj/item/gunbox(src)

```

Comment:
NYET.

Gun box contains _guns_. We just got through discussing why we ultimately decided not to give the detective guns as default equipment.

Gun box also does not contain tasers stop.